plugins/optimization-detective/detect.js

```diff
@@ -722,6 +722,7 @@ export default async function detect( {
 		urlMetric.elements.push( elementData );
 		elementsByXPath.set( elementData.xpath, elementData );
 	}
+	breadcrumbedElementsMap.clear();
```

Comment:
This didn't make sense at the end of the function. So I moved it up here after it is last used.